Lib/test/test_dataclasses.py

```diff
@@ -3214,6 +3214,33 @@ def __post_init__(self, y):
         c = replace(c, x=3, y=5)
         self.assertEqual(c.x, 15)
 
+    def test_initvar_with_default_value(self):
```

Comment:
I find this test extremely confusing. I think most of the tests should be of the form `self.assertEqual(replace(c, y=4), C(1, 2, 3))`. In other words, all of the fields should be tested.